workshops/templates/workshops/all_memberships.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_nav_fluid_sidebar.html" %}
+
+{% load pagination %}
+
+{% block content %}
+    {% if perms.workshops.add_membership %}
+    <p><a href="{% url 'membership_add' %}" class="btn btn-success">New membership</a></p>
+    {% else %}
+    <p><a href="{% url 'membership_add' %}" class="btn btn-success disabled">New membership</a></p>
+    {% endif %}
+{% if all_memberships %}
+    <table class="table table-striped">
+        <tr>
+            <th>organization</th>
+            <th>kind</th>
+            <th>start date</th>
+            <th>end date</th>
+	    <th>contribution</th>
+        </tr>
+    {% for membership in all_memberships %}
+        <tr>
+            <td><a href="{% url 'organization_details' membership.organization.domain %}">{{ membership.organization.fullname }}</a></td>
+	    <td>{{ membership.variant }}</td>
+	    <td>{{ membership.agreement_start }}</td>
+	    <td>{{ membership.agreement_end }}</td>
+	    <td>{{ membership.contribution_type }}</td>
+        </tr>
+    {% endfor %}
+    </table>
```

Comment:
Please also add actions buttons to allow access to `membership_detail` and `membership_edit` views.